client/packages/host/webpack.config.js

```diff
@@ -11,7 +11,7 @@ const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
 
 module.exports = env => {
   const isProduction = !!env.production;
-
+  console.log(env, env.API_HOST)
```

Comment:
dev code? ( i.e. can be removed? )

Reply:
thanks: 7126813dc17146be50f56b3b8598df5403efb3b8